chart/values.yaml

```
@@ -27,7 +27,7 @@ cache:
   # name of frontend service
   # frontendService: kube-httpcache-headless
   # name of backend service
-  backendService: backend-service
+  backendService: "{{ .Release.Name }}-backend-service"
```

Currently, the backend service is not managed by this helm chart -- our assumption was that the upstream application (including the `Service` resource) would usually be managed by a different Helm chart. I'm not opposed to supporting templates in this values (which would in itself even be backwards-compatible), but defaulting this to a value prefixed with the release name might IMO be misleading to an unsuspecting user.

Also, on the (admittedly small) chance that someone deployed this chart with the default `backendService` value, this change would be definitely breaking.

```suggestion
  backendService: "backend-service"
```

---

Ok, I've put the previous default back.
I use this chart as a helm dependency, this allow me to deploy Varnish for each pull request.